manifests/a/Autodesk/AutoCAD2024/24.3/Autodesk.AutoCAD2024.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Autodesk.AutoCAD2024

```

```suggestion
PackageIdentifier: Autodesk.AutoCAD.2024
```
After changing the package identifier, you will also need to change the path of the manifests to `./manifests/a/Autodesk/AutoCAD/2024/24.3.61.0/*`